pkg/pmm/container_test.go

```diff
@@ -61,4 +66,66 @@ func TestContainer(t *testing.T) {
 	for _, env := range expectedEnvs {
 		assert.True(t, foundEnv[env])
 	}
+
+	assert.NotNil(t, container.Lifecycle)
+	assert.Equal(t, []string{"bash", "-c", "pmm-admin unregister --force"}, container.Lifecycle.PreStop.Exec.Command)
+
+	assert.NotNil(t, container.LivenessProbe)
+	assert.Equal(t, intstr.FromInt32(7777), container.LivenessProbe.HTTPGet.Port)
+	assert.Equal(t, "/local/Status", container.LivenessProbe.HTTPGet.Path)
+}
+
+func TestContainer_CustomProbes(t *testing.T) {
+	lp := &corev1.Probe{
+		InitialDelaySeconds: 15,
+		TimeoutSeconds:      7,
+		PeriodSeconds:       11,
+		SuccessThreshold:    1,
+		FailureThreshold:    5,
+	}
+	rp := &corev1.Probe{
+		InitialDelaySeconds: 5,
+		TimeoutSeconds:      4,
+		PeriodSeconds:       12,
+		SuccessThreshold:    1,
+		FailureThreshold:    3,
+	}
+
+	cr := &apiv1alpha1.PerconaServerMySQL{
+		ObjectMeta: metav1.ObjectMeta{Name: "test-cluster"},
+		Spec: apiv1alpha1.PerconaServerMySQLSpec{
+			CRVersion: version.Version(),
+			PMM: &apiv1alpha1.PMMSpec{
+				Image:           "percona/pmm-client:latest",
+				ImagePullPolicy: corev1.PullIfNotPresent,
+				ServerHost:      "pmm-server",
+				LivenessProbes:  lp,
+				ReadinessProbes: rp,
+			},
+		},
+	}
+
+	secret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{Name: "test-secret"},
+		Data: map[string][]byte{
+			string(apiv1alpha1.UserPMMServerToken): []byte("token"),
+			string(apiv1alpha1.UserMonitor):        []byte("monitor-pass"),
+		},
+	}
+
+	c := Container(cr, secret, "mysql", "")
+
+	assert.Equal(t, int32(15), c.LivenessProbe.InitialDelaySeconds)
+	assert.Equal(t, int32(7), c.LivenessProbe.TimeoutSeconds)
+	assert.Equal(t, int32(11), c.LivenessProbe.PeriodSeconds)
+	assert.Equal(t, intstr.FromInt32(7777), c.LivenessProbe.HTTPGet.Port)
+	assert.Equal(t, "/local/Status", c.LivenessProbe.HTTPGet.Path)
+
+	if assert.NotNil(t, c.ReadinessProbe) && assert.NotNil(t, c.ReadinessProbe.HTTPGet) {
```

Comment:
Based on this comment: https://github.com/percona/percona-server-mysql-operator/pull/990#discussion_r2298201834, why we fixed only Liveness probes and not readiness? Isn't it the same?